packages/node-sdk/src/apis/DefaultApi.ts

```
@@ -813,4 +888,61 @@
+        "upsertTargetsRequest",
+        'Required parameter "upsertTargetsRequest" was null or undefined when calling upsertTargets().',
+      );
+    }
+
+    const queryParameters: any = {};
+
+    const headerParameters: runtime.HTTPHeaders = {};
+
+    headerParameters["Content-Type"] = "application/json";
+
+    if (this.configuration && this.configuration.apiKey) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)